data/json/items/tool/science.json

```diff
@@ -1356,5 +1356,100 @@
     "symbol": ";",
     "color": "light_gray",
     "use_action": [ "LUX_METER" ]
+  },
+  {
+    "id": "migo_bio_tech",
+    "type": "TOOL",
+    "name": { "str_sp": "Mi-go Biotech" },
+    "conditional_names": [
+      { "type": "VAR", "condition": "npctalk_var_DISPLAY_NAME_MORALE", "name": { "str_sp": "%s (happy)" } },
+      { "type": "VAR", "condition": "npctalk_var_DISPLAY_NAME_FOCUS", "name": { "str_sp": "%s (focus)" } },
+      { "type": "VAR", "condition": "npctalk_var_DISPLAY_NAME_PAIN", "name": { "str_sp": "%s (pain relief)" } },
+      { "type": "VAR", "condition": "npctalk_var_DISPLAY_NAME_FATIGUE", "name": { "str_sp": "%s (wake up)" } }
+    ],
+    "description": "A piece of mi-go biotechnology.",
+    "weight": "1 g",
+    "volume": "250 ml",
+    "price": 3000,
+    "price_postapoc": 10000,
+    "symbol": "!",
+    "color": "light_red",
+    "use_action": {
+      "type": "effect_on_conditions",
+      "description": "Touch the opening.",
+      "effect_on_conditions": [
+        {
+          "id": "EOC_MIGO_BIO_TECH_INTEREST",
+          "condition": { "not": { "one_in_chance": 3 } },
+          "effect": [
+            { "message": "You have a slight feeling of being watched." },
```

Comment:
I am pretty sure it should be `u_message`, not `message`.

Reply:
This is exactly what's causing #50590.